pulpcore/pulpcore/app/viewsets/base.py

```diff
@@ -81,6 +81,32 @@ class NamedModelViewSet(viewsets.GenericViewSet):
     parent_lookup_kwargs = {}
     schema = DefaultSchema()
 
+    def get_serializer_class(self):
+        """
+        Fetch the serializer class to use for the request.
+
+        The default behavior is to use the "serializer_class" attribute on the viewset.
+        We override that for the case where a "minimal_serializer_class" attribute is defined
+        and where the request contains a query parameter of "pretty=True".
+
+        The intention is that ViewSets can define a second, more minimal serializer with only
+        the most important fields.
+        """
+        assert self.serializer_class is not None, (_(
+            "'{}' should either include a `serializer_class` attribute, or override the "
+            "`get_serializer_class()` method.".format(self.__class__.__name__)
+        ))
+        minimal_serializer_class = getattr(self, 'minimal_serializer_class', None)
+
+        if minimal_serializer_class:
+            if 'pretty' in self.request.query_params:
```

Comment:
I like the name "pretty" but apparently it's often used to mean "JSON pretty print" with readable nesting.  Should it be changed?

Reply:
The best alternate is 'minimal' I think.

Reply:
how about `core`?

Reply:
```core``` would also be a good choice